todo-cli.cabal

```diff
@@ -1,54 +1,38 @@
--- This file has been generated from package.yaml by hpack version 0.20.0.
---
--- see: https://github.com/sol/hpack
---
--- hash: 592e31a8bba4a31e2540e7aea82b3da47430f4c1a8a18f898de13cd3be53b9a9
-
-name:           todo-cli
-version:        0.1.0.0
-description:    Please see the README on Github at <https://github.com/StasyaGera/todo-cli#readme>
-homepage:       https://github.com/StasyaGera/todo-cli#readme
-bug-reports:    https://github.com/StasyaGera/todo-cli/issues
-author:         Stanislava Gera
-maintainer:     stasya.gera@yandex.ru
-copyright:      2018 Stanislava Gera
-license:        BSD3
-license-file:   LICENSE
-build-type:     Simple
-cabal-version:  >= 1.10
-
-extra-source-files:
-    ChangeLog.md
-    README.md
+name:                   todo-cli
+version:                0.1.0.0
+description:            Please see the README on Github at <https://github.com/StasyaGera/todo-cli#readme>
+homepage:               https://github.com/StasyaGera/todo-cli#readme
+bug-reports:            https://github.com/StasyaGera/todo-cli/issues
+author:                 Stanislava Gera
+maintainer:             stasya.gera@yandex.ru
+copyright:              2018 Stanislava Gera
+license:                BSD3
+license-file:           LICENSE
+build-type:             Simple
+cabal-version:          >= 1.10
+extra-doc-files:        ChangeLog.md
+                      , README.md
 
 source-repository head
-  type: git
-  location: https://github.com/StasyaGera/todo-cli
+  type:                 git
+  location:             https://github.com/StasyaGera/todo-cli
 
 library
-  hs-source-dirs:
-      src
-  build-depends:
-      base >= 4.9 && < 5
-  ghc-options:
-      -Wall
-  exposed-modules:
-      Todo.Model
-  other-modules:
-      Paths_todo_cli
-  default-language: 
-      Haskell2010
-  default-extensions: RecordWildCards
+  hs-source-dirs:       src
+  build-depends:        base >= 4.9 && < 5
+  ghc-options:          -Wall
+  exposed-modules:      Todo.Model
+  other-modules:        Paths_todo_cli
+  default-language:     Haskell2010
+  default-extensions:   RecordWildCards
+                      , GeneralizedNewtypeDeriving
```

Comment:
`,` is not needed in `default-extensions`. But this is not important for now, you can remove it when you add another extension here.